### ISSUE TITLE
There is an error when the backend starts

### ISSUE THREAD

**Issue by kkml** [kubevious/deploy#1]
Here is the error I get when I try to install the chart:

```
[2020-03-02 14:22:58.037 +0000] INFO  (LogicProcessor/1 on kubevious-655b67b8f5-8f8dn): [process] BEGIN
/app/lib/logic/parsers/035_autoscaler.js:32
        var appProps = appScope.properties;
                                ^

TypeError: Cannot read property 'properties' of undefined
    at Object.handler (/app/lib/logic/parsers/035_autoscaler.js:32:33)
    at LogicProcessor._processHandler (/app/lib/logic/processor.js:231:21)
    at LogicProcessor._processParser (/app/lib/logic/processor.js:177:18)
    at LogicProcessor._processParsers (/app/lib/logic/processor.js:163:18)
    at LogicProcessor.process (/app/lib/logic/processor.js:144:14)
    at FacadeRegistry._handleConcreteRegistryChange (/app/lib/facade/registry.js:111:42)
    at FacadeRegistry.handleAreLoadersReadyChange (/app/lib/facade/registry.js:120:14)
    at Context._checkLoadersReady (/app/lib/context.js:129:37)
    at K8sLoader.readyHandler [as _readyHandler] (/app/lib/context.js:77:22)
    at K8sLoader._reportReady (/app/lib/loaders/k8s.js:150:14)
    at Timeout._onTimeout (/app/lib/loaders/k8s.js:68:18)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
``
`

**Comment by rubenhak** [comment]
@kkml, could you provide some more details from your setup? Looks like there is a HorizontalPodAutoscaler used in your cluster. Could you check what its' **spec.scaleTargetRef** is referring to? Is it Deployment, StatefulSet, ... ?

**Comment by rubenhak** [comment]
@kkml, please provide version tag for **kubevious** deployment.

**Comment by rubenhak** [comment]
@kkml, Please try latest build following instructions here: https://github.com/kubevious/deploy#deploying-latest-build

**Comment by rubenhak** [comment]
The fix is incorporated to v0.3

**Comment by kkml** [comment]
@rubenhak works like a charm, thanks ;)
